src/CommunityToolkit/Xamarin.CommunityToolkit/Views/MediaElement/Android/FormsVideoView.android.cs

```diff
@@ -55,21 +56,15 @@ public override void SetVideoURI(global::Android.Net.Uri uri, IDictionary<string
 			base.SetVideoURI(uri, headers);
 		}
 
-		public override void SetVideoURI(global::Android.Net.Uri uri)
-		{
-			GetMetaData(uri, new Dictionary<string, string>());
-			base.SetVideoURI(uri);
-		}
-
```

Comment:
Don't we need GetMetaData on setting URI anymore? Can you please explain why do we remove it. I am a little out of context :)

Reply:
the base.SetVideoURI(Uri) calls SetVideoUri(Uri,IDictionary<string,string>) so we only need to override that one method otherwise GetMetaData is called twice for each Uri source.